README.md

```diff
@@ -157,9 +157,11 @@ cargo dist init
 If you want to just (re)generate the ci scripts, you can do:
 
 ```sh
-cargo dist generate-ci github
+cargo dist generate-ci
 ```
 
+(Note that for this precise invocation you should have `ci = ["github"]` set in your Cargo.toml)
```

Comment:
if folks don't have ci = github in their cargo.toml what happens

Reply:
Added more text basically saying "hmm maybe the cli flags for this are Bad now"